### DISCUSSION TITLE
button meaning

### DISCUSSION THREAD

**Discussion by Martinligabue** [TrianguloY/URLCheck#262] [Q&A] [answered]
Hi, what is the button, when opening a link, in this app, in the bottom left side, that says "enable/disable" when long pressed? I don't get what it enables.

**Comment by Murilogs1910** [comment, marked as answer]
Could you share a screenshot indicating the button? I'm not aware of any that shows "enable/disable".

EDIT: Actually, is your app in Italian? That entry was incorrectly translated, apparently. If that is the case, then I believe you are referring to the button circled in the screenshot 
![image11](https://github.com/TrianguloY/UrlChecker/assets/86175855/f4be21d3-2367-4392-8ebf-2f515249ab57)
That button is used to enable/disable the custom tabs.

**Reply by Martinligabue** [reply]
yes, that's the one, I assumed it was correctly translated, thanks for the explanation.
I don't think I have custom tabs in my phone, since it didn't do anything either way

**Reply by Murilogs1910** [reply]
Are you sure you don't have custom tabs? That feature comes with the browser, and I'm not aware of any browser without that. May I know which browser you are using?

**Reply by TrianguloY** [reply]
~~Maybe Mull doesn't have it.~~ wait, that screenshot is from you 😅 so yes, what Murilogs said, which browser do you have? And which android version? 

These are custom tabs: https://developer.chrome.com/docs/android/custom-tabs/

If it does nothing for you, you can hide it from the module settings

As for the Italian translation, I'll mark it as needs work, hopefully someone can translate it properly: https://hosted.weblate.org/translate/urlcheck/app-strings/it/?checksum=fc914a99edfa9a0f
